### DISCUSSION TITLE
Problem open the astroidframe.work site

### DISCUSSION THREAD

**Discussion by sonvnn** [templaza/astroid-framework#679] [General]
Hello everyone,

A few days ago I received an email from a user reporting not being able to access astroidframe.work. Can anyone help me get to astroidframe.work? And please let me know if you can't access the site for anything.

Thanks & Best Regards,
Sonny

**Comment by Chacapamac** [comment]
Just access —> https://astroidframe.work/ without problems.
No console errors in any of the pages.

The only little issue I see is that submenus are position to the left oft the main menu you click.
![Screen Shot 2024-05-14 at 9 19 57 AM](https://github.com/templaza/astroid-framework/assets/1907276/37b2cf17-d851-4f9a-992a-f12bff8271e0)

**Good Day** @sonvnn

**Reply by sonvnn** [reply]
Thank you @Chacapamac. I fixed it

**Comment by vtheod** [comment]
No problem with accessing the site

**Comment by Dnsftl** [comment]
No problem reaching https://astroidframe.work/, Everything works properly here

**Comment by rinka88** [comment]
Or maybe this is a problem of a certain country or telecom operator. we need to clarify first.

**Comment by Banuk1616** [comment]
Apart from the constant warning from Bitdefender, I have never had any problems accessing the Astroid website.

**Comment by Chacapamac** [comment]
`constant warning from Bitdefender`  This is a problem report before —> https://github.com/templaza/astroid-framework/discussions/411
@Banuk1616 can you specify the Bitdefender message or warning?
